docs/source/get_involved/code_of_conduct.rst

```diff
@@ -133,15 +159,50 @@ project event or platform include:
 -  nonconsensual or unwelcome physical contact
 -  sustained disruption of talks, events or communications
 -  insults or put downs
--  sexist, racist, homophobic, transphobic, ableist, or exclusionary
-   jokes
+-  sexist, racist, homophobic, transphobic, ableist, or exclusionary jokes
 -  excessive swearing
 -  incitement to violence, suicide, or self-harm
 -  continuing to initiate interaction (including photography or
    recording) with someone after being asked to stop
 -  publication of private communication without consent
+-  decision making on resource allocation which has the effect of excluding people
+   on the basis of protected characteristics, or other forms of systemic discrimination
+-  decision making on resource allocation that confers advantages to personal networks,
```

Comment:
On the topic of resource allocation I was wondering - does `sktime` have any sort of formal budget?  I was unable to find such a document, but wonder if it could be a good thing to put into place going forward! :) Reasons being:
- It can be helpful in securing future funding if you show evidence of where money has been spent in the past/how.
- Can help the community achieve certain goals if where the money will be spent is planned/discussed. 
- Is often helpful in reducing conflict in advance if such a document is in place already!

Could be something the community could get together and discuss once or twice a year?  Could also be an opportunity for brainstorming and/or grant writing sessions? 

Just chiming in on this as I know this year `sktime` paid for the dev-days and helped the GSoC interns come! (Which I am of course biased in favor of having been a good idea!) and I think having some sort of paper trail that tracks these sorts of activities can be helpful.  Would other people find this a useful exercise going forward?  (Or does it perhaps already exist and I have missed it?)

Reply:
I am generaly in favour of more open-ness and having some kind of record/paper trail.

Regarding budget/ This page has some info on funding: https://www.sktime.org/en/stable/about/funding.html.  i didn't know this, but just saw there is a link to where you can donate to sktime: https://opencollective.com/sktime.  and then if you explore, you see how the opencollective funds are spent: https://opencollective.com/sktime#category-BUDGET

Reply:
> does sktime have any sort of formal budget?

yes, and no, multiple things:

On status:
* `sktime` is not a formal legal entity, so all budgets are currently "on behalf of the project", but the formal status is usually at different places, and accountability is with different individuals affiliated with `sktime`
* we are currently affiliated with NumFOCUS and exploring to move to sponsored status. That would give `sktime` a status as a formal legal entity (part of umbrella NumFOCUS), and NumFOCUS would become the budget administering entity
* the Alan Turing Institute is not related to `sktime` status, that we are here has more historical reasons

On de-facto budgets:
* `sktime` has an "open collective" account where we currently administer donations and reimbursements https://opencollective.com/sktime, this is fully open.
* `sktime` has recently won an UKRI grant of approx 500.000 GBP volume: https://gow.epsrc.ukri.org/NGBOViewGrant.aspx?GrantRef=EP/W030756/1. This is administered by individuals at UEA, one of our partner universities. Said individuals are `sktime` affiliated, but formally have full control and do not need to (and choose not to) provide transparent views of what the money is being spent on. `sktime` itself is not a formal entity, and not an independent research organisation, so cannot apply for grants. I think this mismatch of accountability/authority is a fatal construction flaw in the UKRI grant call that was meant to support open source communities. Similar issues exist with funds sourced from US and continental European government research funding agancies.
* there are further accounts that individuals administer to the benefit, or on behalf of `sktime`. For instance, I have a small pot of emergency funds reserved for `sktime` at UCL, some funds that originate from my consultancy work 5 years ago and which I can use at my discretion to the benefit of `sktime` (or any other academic activities).
* as above, the Alan Turing Institute does not offer any de-facto assistance to `sktime`. Despite us being here (and figuring in public facing presentation on the great work the Turing is doing), we are not receiving any benefits from the Turing. We are also not involved or benefitting from the substantial funding pots allocated to "open research community building", see here: https://github.com/alan-turing-institute/open-community-building. Personally, I find that strange, but formally the Turing do not *need* to support `sktime` in any way, and they choose not to.

Reply:
so, from the perspective of code of conduct, we need to set out expectations for a complex system of individual stakeholders, `sktime` affiliated.
Some might get awarded benefits and resources due to achievements of the `sktime` commnunity, and, once awarded, typically have almost absolute authority on decision making on said resources. Some others might choose to donate to `sktime` or award benefits to the community, but said benefits may benefit some parts of the community and not others.